src/code/z_common_data.c

```diff
@@ -24,4 +24,7 @@ void SaveContext_Init(void) {
     gSaveContext.dogIsLost = true;
     gSaveContext.nextTransitionType = TRANS_NEXT_TYPE_DEFAULT;
     gSaveContext.prevHudVisibilityMode = HUD_VISIBILITY_ALL;
+#if OOT_NTSC
+    gSaveContext.language = LANGUAGE_ENG;
+#endif
```

Comment:
I assume this check will make more sense at a later time, when we start differentiating between eng and jp?

Reply:
Yeah this is what it will be for all GC retail versions at least (not sure the best way to write it): https://github.com/cadmic/oot/blob/c6f0f4a7ce4a87fa016d2241e93c1b6567681d06/src/code/z_common_data.c#L27